resource/subfunctions/started_task.md

```diff
@@ -0,0 +1,276 @@
+
+#### 📥 Parameters
+* `started_task_name`<br>
+  The name of the **Started Task** profile being defined to the **STARTED** class.
+
+* `traits`<br>
+  A dictionary of **traits/attributes** that should be given to the resource on creation. See [Traits](../../base/segments_traits_operators#traits) to see what all of the valid **Resource Traits** are.
+
+#### 📤 Returns
+* `Union[dict, bytes]`<br>
+  Returns a **Security Result dictionary** or **Security Request XML bytes** if the `ResourceAdmin.generate_requests_only` class attribute is set to `True`.
+
+#### ❌ Raises
+* `SecurityRequestError`<br>
+  Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `AlterOperationError`<br>
+  Raises `AlterOperationError` when the **z/OS userid** supplied cannot be altered because it does **NOT** exist.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** provided contains one or more **unknown** traits.
+
+#### 💻 Example
+
+###### Python REPL
+```python
+from pyracf import ResourceAdmin
+resource_admin = ResourceAdmin()
+resource_admin.alter_started_task("TSTTSKEL",traits={"stdata:trusted": True})
+{'securityResult': {'resource': {'name': 'TSTTSKEL', 'class': 'STARTED', 'operation': 'set', 'requestId': 'ResourceRequest', 'info': ['Definition exists. Add command skipped due  to precheck option'], 'commands': [{'safReturnCode': 0, 'returnCode': 0, 'reasonCode': 0, 'image': 'RALTER  STARTED             (TSTTSKEL)  STDATA   (TRUSTED     (YES))', 'messages': ['IRR52148I Warning: A value for USER should be specified in STDATA.', 'IRR52149I Warning: STARTED profiles should have (or match) names with two qualifiers.', 'ICH11009I RACLISTED PROFILES FOR STARTED WILL NOT REFLECT THE UPDATE(S) UNTIL A SETROPTS REFRESH IS ISSUED.']}]}, 'returnCode': 0, 'reasonCode': 0}}
+```
+
+###### Security Result Dictionary as JSON
+```json
+{
+  "securityResult":{
+    "resource":{
+      "name":"TSTTSKEL",
+      "class":"STARTED",
+      "operation":"set",
+      "requestId":"ResourceRequest",
+      "info":[
+        "Definition exists. Add command skipped due  to precheck option"
+      ],
+      "commands":[
+        {
+          "safReturnCode":0,
+          "returnCode":0,
+          "reasonCode":0,
+          "image":"RALTER  STARTED             (TSTTSKEL)  STDATA   (TRUSTED     (YES))",
+          "messages":[
+            "IRR52148I Warning: A value for USER should be specified in STDATA.",
+            "IRR52149I Warning: STARTED profiles should have (or match) names with two qualifiers.",
+            "ICH11009I RACLISTED PROFILES FOR STARTED WILL NOT REFLECT THE UPDATE(S) UNTIL A SETROPTS REFRESH IS ISSUED."
+          ]
+        }
+      ]
+    },
+    "returnCode":0,
+    "reasonCode":0
+  }
+}
+```
+
+## `ResourceAdmin.extract_started_task()`
+
+```python
+  def extract_started_task( self, started_task_name: str ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def extract_started_task(self, started_task_name: str) -> Union[dict, bytes]:
```

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.